.github/workflows/ci.yml

```diff
@@ -123,6 +123,72 @@ jobs:
       - run: make man
         working-directory: src/github.com/containerd/containerd
 
+  # Make sure binaries compile with other platforms
+  crossbuild:
+    name: Crossbuild Binaries
+    needs: [project, linters, protos, man]
+    runs-on: ubuntu-18.04
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - goos: linux
+            goarch: arm64
+          - goos: linux
+            goarch: arm
+            goarm: "7"
+          - goos: freebsd
+            goarch: amd64
+
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.16.2'
+      - name: Set env
+        shell: bash
+        run: |
+          echo "GOPATH=${{ github.workspace }}" >> $GITHUB_ENV
+          echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
+      - uses: actions/checkout@v2
+        with:
+          path: src/github.com/containerd/containerd
+      - run: |
+          set -e -x
+
+          packages=""
+          platform="${{matrix.goos}}/${{matrix.goarch}}"
+          if [ -n "${{matrix.goarm}}" ]; then
+            platform+="/v${{matrix.goarm}}"
+          fi
+
+          case "${platform}" in
+          linux/arm/v7)
+            packages+=" crossbuild-essential-armhf"
+            echo "CGO_ENABLED=1" >> $GITHUB_ENV
+            echo "CC=arm-linux-gnueabihf-gcc" >> $GITHUB_ENV
```

Comment:
Where do these toolchains come from?

Reply:
From `crossbuild-essential-<arch>`

Reply:
Neat, I had no idea that existed!

Reply:
core Ubuntu packages: e.g. crossbuild-essential-armhf

Reply:
thank you GitHub refresh :)

Reply:
armv7 would be a useful default to cover RPi 2-4. If you can manage to cover armv6 then it would allow use of the Raspberry Pi 1 and Zero, but both are extremely slow and not well suited to containers (my opinion). 

Looking forward to deprecating the community arm builds of containerd for faasd users.

Reply:
Only adding these builds to make sure the build doesn't break as happened with zfs recently and a couple of different issues with Windows.